volatility3/cli/text_renderer.py

```diff
@@ -25,7 +25,7 @@
     vollog.debug("Disassembly library capstone not found")
 
 
-def hex_bytes_as_text(value: bytes) -> str:
+def hex_bytes_as_text(value: bytes, width: int = 16) -> str:
```

Comment:
I'm not sure this parameter will ever really be usable in any instances, but I'm happy leaving it in there, it's a reasonable place to store a constant.